File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.Arrays;

class Solution {
    boolean solution(String s) {
        long pCount = Arrays.stream(s.split("")).map(String::toLowerCase).filter(i -> i.equals("p")).count();
        long yCount = Arrays.stream(s.split("")).map(String::toLowerCase).filter(i -> i.equals("y")).count();
        return pCount == yCount;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{   
    string str = "";
    for(char ch : s) {
        str += tolower(ch);
    }
    int count = 0;
    for(int i = 0; i < s.length(); i++) {
        if(str.at(i) == 'p') {
            count++;
        } else if(str.at(i) == 'y') {
            count--;
        }
    }
    return count == 0;
}